src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,19 @@
-const humanCatDogYears = number => {};
+const humanCatDogYears = number => {
+  if (number === 1) {
+    catYear = 15;
+    dogYear - 15
```

Comment:
may be a typo here. It's also best practice to define your variables in the uppermost scope of the function with either a const or a let. A let would be good here.